### PR TITLE
Modify css for better web accessibility

### LINKED ISSUE #89: Problems with dropdown list
When the dropdown list appears, the button is barely visible:

image

We should try and adapt the CSS to make it more visible:

image

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
This PR mainly focusing on:
- [x] Change the font size and color of main heading
- [x] Reduce the column 1 (column with search bar button)
- [x] Change the code block color back to ``pink``
- [x] dropdown list button visibility. Fix  #89 
ansys-sphinx-theme-new1
ansys-sphinx-theme-new2
ansys-sphinx-theme-new3

### CODE REVIEW

**Comment by jorgepiloto** [comment]
Thanks for addressing this, @Revathyvenugopal162. Let me ping here @timdansys so he is aware about the changes we introduce regarding accessibility.

Tim is still working on his notes and will share those in the incoming days.

**Comment by jorgepiloto** [comment]
From our private discussion, I think that inline code snippets (i.e. anything between ``)  should also have a gray background. See [this example in pydata-sphinx-theme](https://pydata-sphinx-theme.readthedocs.io/en/latest/community/setup.html#clone-the-repository).

**Comment by PipKat** [comment]
@Revathyvenugopal162 Your changes look good to me. I'll let Jorge, Maxime, or Roberto be the approver though. The one thing I do remember Tim talking about is our possibly using gray boxes like this for inline code (literal tagging): 
image. 
However, I went to https://new.scipy.org/ to see what the doc for all the big scientific packages looked like. Sphinx was used for all but the IPthon doc, which was far less attractive and usable. All the the others used Sphinx, and of those, SymPy was the only one that didn't. However, it used a custom-branded green color scheme that isn't appealing to me at all!

**Comment by PipKat** [comment]
> From our private discussion, I think that inline code snippets (i.e. anything between ``) should also have a gray background. See [this example in pydata-sphinx-theme](https://pydata-sphinx-theme.readthedocs.io/en/latest/community/setup.html#clone-the-repository).

Jorge's suggestion is a great compromise. I think the pinkish-red text is easier to read in gray highlighted box than the black text.

**Comment by Revathyvenugopal162** [comment]
Added the inline code gray background.Thanks @PipKat and @jorgepiloto for pointing this

File: src/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css [hunk 1]
```diff
@@ -442,3 +457,88 @@ a > code.download
...
+.navbar button.navbar-toggler {
+    margin-right: 1em;
+    border-color: rgb(255, 255, 255);
+    color: rgb(255, 255, 255);
+}
+
+/*
+###################################################
+Side colomn size(first and second colomn from left)
+###################################################
```
Comment (RobPasMue) [inline src/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css:486]:
```suggestion
####################################################
Side column size (first and second column from left)
####################################################
```

File: src/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css [hunk 2]
```diff
@@ -442,3 +457,88 @@ a > code.download
...
+
+.toc-entry a:hover {
+  color: var(--pst-color-link);
+  text-decoration: none;
+}
+
+/*
+###########################################
+Navigation colomn(according to side colomn)
+###########################################
```
Comment (RobPasMue) [inline src/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css:537]:
```suggestion
############################################
Navigation column (according to side column)
############################################
```

**Review by RobPasMue — APPROVED** [review]
LGTM apart from the mentioned comments and @jorgepiloto's and @PipKat's suggestions. Can you update the screenshots to the latest versions after your changes?

**Comment by RobPasMue** [comment]
screenshot

By the way, I thought we were going to change the color and way of showing the top bar menus when hovering or accessing (e.g. using an underline). Will this be handled in this PR @Revathyvenugopal162?

**Comment by jorgepiloto** [comment]
I think we can implement the "chapter underline" in a separate pull request, @RobPasMue @Revathyvenugopal162.

Also, @timdansys still needs to share with use his final notes about the new style. I would be nice to implement the proposed changes in issues to have a better idea on what has been implemented so far.

File: src/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css [hunk 3]
```diff
@@ -442,3 +457,88 @@ a > code.download
...
+/*
+###############
+Dropdown button
+###############
+*/
+
+.navbar button.navbar-toggler {
+    margin-right: 1em;
+    border-color: rgb(255, 255, 255);
+    color: rgb(255, 255, 255);
```
Comment (MaxJPRey) [inline src/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css:480]: @Revathyvenugopal162 Thanks for adding it. 👍

**Review by MaxJPRey — APPROVED** [review]

**Comment by MaxJPRey** [comment]
> From our private discussion, I think that inline code snippets (i.e. anything between ``) should also have a gray background. See [this example in pydata-sphinx-theme](https://pydata-sphinx-theme.readthedocs.io/en/latest/community/setup.html#clone-the-repository).

@jorgepiloto I agree, the contrast is better than for [numpy](https://numpy.org/doc/1.23/user/absolute_beginners.html)

For example:
image

**Comment by Revathyvenugopal162** [comment]
code block
Adding the code inline boxes image.

**Review by jorgepiloto — APPROVED** [review]

**Comment by da1910** [comment]
I'm not entirely sure if it's worth making a new issue for this, but if we're thinking about accessibility - the contrast for the blue links, pink code/quote sections, and orange syntax highlighting in light mode is below the recommended value:

image


image

image

The WCAG guidelines - https://www.w3.org/WAI/standards-guidelines/wcag/ are probably worth following in this case, and aiming for `AA` level is quite reasonable (not much change needs to be made), aiming for `AAA` would be a much more obvious change.

Below is the suggested minimum contrast level in light mode:
image

Links - #0077B3
Quotes - #CD186D
Syntax - #B35000

**Comment by Revathyvenugopal162** [comment]
> 

Thanks @da1910 for the color values. Adding these values.

File: src/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css [hunk 4]
```diff
@@ -116,7 +116,7 @@ html[data-theme="dark"] {
   * layout
   */
 
-  --pst-color-link: #0088CC;
+  --pst-color-link: #0077B3;
```
Comment (da1910) [inline src/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css:119]: We shouldn't change this, #0077B3 also fails accessibility guidelines in dark mode.

**Review by da1910 — APPROVED** [review]
Looks good for contrast now!

**Comment by Revathyvenugopal162** [comment]
Merging. Thanks @MaxJPRey @jorgepiloto @RobPasMue @PipKat @da1910 for the thorough review.

**Pull request merged**